src/components/Filter.svelte

```diff
@@ -6,6 +6,11 @@
 
     let filter: string = '';
     let filterValid: boolean = false;
+    let focus: boolean = false;
+
+    function handleFocus() {
```

Comment:
For very simple handlers you can also define them [inline](https://svelte.dev/tutorial/inline-handlers)